src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +25,25 @@
      *                 В любом другом случае бросить DateTimeException
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        LocalDate date = LocalDate.now();
+        switch (datePart) {
+            case DAY: {
+                return date.getDayOfMonth() + "";
```

Comment:
```suggestion
                return String.valueOf(date.getDayOfMonth());
```
same everywhere inside this method